### PR TITLE
fixes #264 - Increased map space in CountryTweetMap app

### LINKED ISSUE #264: Change CountryTweetMap app's layout to provide more space to map
Presently map is not given much space. Tweak the layout to get some more space for the map.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #264, added side bar and removed nav bar to provide more
space to map.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/CountryTweetMap/

@kavithaenair @Achint08 @SKrPl @singhpratyush open for review

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/CountryTweetMap/

@kavithaenair @Achint08 @SKrPl @singhpratyush open for review

File: CountryTweetMap/css/styles.css [hunk 1]
```diff
@@ -175,6 +175,90 @@ body {
...
+.sidenav {
+    height: 100%;
+    width: 0;
+    position: fixed;
+    z-index: 1001;
+    top: 0;
+    left: 0;
+    background-color: #fafafa;
+    overflow-x: hidden;
+    transition: width 0.5s;
```
Comment (Achint08) [inline CountryTweetMap/css/styles.css]: Can this be `transform` instead of `width`?

File: CountryTweetMap/css/styles.css [hunk 2]
```diff
@@ -175,6 +175,90 @@ body {
...
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+.sidebar-header {
+    background-color: #D81B60;
+    padding-left: 10px;
```
Comment (Achint08) [inline CountryTweetMap/css/styles.css]: Specify padding in one line using `padding: top right bottom left`.

**Comment by djmgit** [comment]
@Achint08 used transform

**Pull request merged**